### DISCUSSION TITLE
.VCE files?

### DISCUSSION THREAD

**Discussion by StariusPrime** [gtoal/pitrex#7] [Q&A]
Hello!  
I noticed that all the roms on the PiTrex image are .bin files.  My collection of roms are primarily .vce files.  Can the PiTrex natively use these?   Can .vce be converted to .bin if not?

**Comment by gtoal** [comment]
The use of .bin is just an arbitrary extension. They're raw binary files as dumped from the machine. I don't know what vce files are but if they are plain binary then all you would need to do is compare the checksums and if they match, rename the files.
By the way, this is not the repo for the stand-alone image - that would be https://github.com/malbanGit/pitrex-baremetal and probably a better place to ask questions would be the mailing list - subscription info at https://groups.io/g/pitrex-dev/

**Reply by StariusPrime** [reply]
Thank you for the clarification! 👍
